codedeploy/be/scripts/start.sh

```
@@ -0,0 +1,10 @@
+#!/bin/bash
+# scripts/start.sh
+
+APP_DIR="/home/ubuntu/app"
```

Shellcheck (0.10.0)

[warning] APP_DIR appears unused. Verify use (or export if used externally).

(SC2034)